.github/workflows/ci.yaml

```
@@ -118,23 +102,37 @@ jobs:
           set -x
           sudo apt-get install -y -qq xvfb ffmpeg mesa-utils fluxbox xserver-xorg xserver-xorg-core xserver-xorg-video-all libwayland-egl1-mesa
           set +x
-          . ${GITHUB_WORKSPACE}/catkin_ws/devel/setup.bash
+          if [ "${{ matrix.catkin-build }}" == "catkin" ]
+          then
+            . ${GITHUB_WORKSPACE}/catkin_ws/devel/setup.bash
+          fi
           set -x
           export DISPLAY=":1"
           Xvfb ${DISPLAY} -screen 0 1920x1080x24 &
           sleep 10s
           fluxbox 2> /dev/null &
           mkdir -p ${HOME}/.config/mc_rtc/controllers
-          cp ${GITHUB_WORKSPACE}/catkin_ws/src/isri-aist/LocomanipController/etc/mc_rtc.yaml ${HOME}/.config/mc_rtc
-          cp ${GITHUB_WORKSPACE}/catkin_ws/src/isri-aist/LocomanipController/.github/workflows/config/${{ matrix.motion-type }}.yaml ${HOME}/.config/mc_rtc/controllers/LocomanipController.yaml
+          if [ "${{ matrix.catkin-build }}" == "catkin" ]
+          then
+            cp ${GITHUB_WORKSPACE}/catkin_ws/src/${{ github.repository }}/etc/mc_rtc.yaml ${HOME}/.config/mc_rtc
```

In the following `depends.rosinstall`, it is described as `isri-aist`, so we should not use `${{ github.repository }}`, right?
```suggestion
            cp ${GITHUB_WORKSPACE}/catkin_ws/src/isri-aist/etc/mc_rtc.yaml ${HOME}/.config/mc_rtc
```
https://github.com/isri-aist/LocomanipController/blob/e7798561383da4dbba47235c882e95948f4b2257/depends.rosinstall#L2

---

This is getting the `mc_rtc.yaml` generated by `LocomanipController` which is not described in `depends.rosinstall` and is cloned by the `build-catkin-project` action in `${GITHUB_WORKSPACE}/catkin_ws/src/${{ github.repository }}` ([here](https://github.com/jrl-umi3218/github-actions/blob/master/build-catkin-project/action.yml#L37))

---

Sorry, it's OK as it is.